pkg/master/master.go

```diff
@@ -188,6 +188,9 @@ func (m *Master) InstallAPIs(c *Config) {
 			Scheme:               api.Scheme,
 			ParameterCodec:       api.ParameterCodec,
 			NegotiatedSerializer: api.Codecs,
+			SubresourceGroupVersionKind: map[string]unversioned.GroupVersionKind{
+				"replicationcontrollers/scale": unversioned.GroupVersion{Group: "autoscaling", Version: "v1", Kind: "Scale"},
```

Comment:
Fixed this.

> I don't know if the case difference matters, but below it is registered as replicationControllers/scale

@liggit No. `replicationControllers` is not right, it should all be lowercase. There is lowercase-coercion going on somewhere, so it works. It should be fixed everywhere, but I don't think this PR is right for that.